### PR TITLE
V2 reconfig tokens tab

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

Adds ability to edit tokens tab (redemp rate, discount rate, reserved rate and receivers). 

Also adds currency symbol to distributionLimit on v2 project (not shown in attached vid)

## Screenshots or screen recordings

https://user-images.githubusercontent.com/96150256/159870206-38a3fb7e-baf9-4899-ad36-27e895d9bc3f.mp4

## Acceptance checklist

- [ ] I have evaluated the
      [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines)
      for this PR.
- [ ] I have tested this PR in
      [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/components/v2/V2Create/tabs/TokenTab/TokenTabContent.tsx [hunk 1]
```diff
@@ -122,7 +126,7 @@ export default function TokenTabContent({ onFinish }: TabContentProps) {
 
   return (
     <Row gutter={32}>
-      <Col md={10} xs={24}>
+      <Col md={!hidePreview ? 10 : 24} xs={24}>
```
Comment (aeolianeth) [inline src/components/v2/V2Create/tabs/TokenTab/TokenTabContent.tsx:129]: we should make a layout component for this stuff eventually to reduce duplication

File: src/components/v2/V2Create/tabs/TokenTab/TokenTabContent.tsx [hunk 2]
```diff
@@ -202,10 +206,11 @@ export default function TokenTabContent({ onFinish }: TabContentProps) {
                 : undefined
             }
           />
-          <FormActionbar />
+          {/* Default to floating save button if custom one not given */}
+          {saveButton ? saveButton : <FormActionbar />}
```
Comment (aeolianeth) [inline src/components/v2/V2Create/tabs/TokenTab/TokenTabContent.tsx:210]:
```suggestion
          {saveButton ?? <FormActionbar />}
```

**Review by aeolianeth — APPROVED** [review]
@johnnyd-eth LGTM, I haven't tested it but if you're confident in it, please go ahead and merge 💪

File: src/components/v2/V2Create/tabs/TokenTab/TokenTabContent.tsx (hunk 1 above)
Comment (johnnyd-eth) [reply src/components/v2/V2Create/tabs/TokenTab/TokenTabContent.tsx:129]: Yeah true that.

**Pull request merged**